### ISSUE TITLE
File Menu opening a new file (Like Ctrl + N) instead of expanding the Menu

### ISSUE THREAD

**Issue by bhosale-ajay** [Microsoft/vscode#61262]
- VSCode Version: 1.28.2 (user setup)
- OS Version: Windows 10

Steps to Reproduce:

1. Open Code
2. Click on "File" Menu
3. Expected - Expand File Menu
4. Actual - Open a new untitled file

![image](https://user-images.githubusercontent.com/1799587/47177748-4a434100-d2e7-11e8-8b9e-be00ae96604c.png)

Does this issue occur when all extensions are disabled?: YES

**Comment by sbatten** [comment]
@bhosale-ajay when you click on the file menu, do you see any animation of the menu appearing at all?

**Comment by bhosale-ajay** [comment]
No, On further observation, this happens with screen zoomed (+4) and when double clicked on File.

**Comment by sbatten** [comment]
@bhosale-ajay even under these circumstances I do not see the behavior you describe. Would you be able to provide a gif of your whole window while reproing the behavior?

**Comment by bhosale-ajay** [comment]
Please check
![filemenu](https://user-images.githubusercontent.com/1799587/47516118-a14e9600-d852-11e8-8d36-0359ea310f4c.gif)

**Comment by sbatten** [comment]
@bhosale-ajay can you verify with the latest version of VS Code?

**Comment by bhosale-ajay** [comment]
Verified, and looks like it is fixed
